config/locales/uk.yml

```diff
@@ -1,6 +1,20 @@
 uk:
+  home: Головна
+  welcome: Вітаємо
+  to_quizland: у Квізленді
+  first_quiz: Створити перший тест
+  about: Про нас
   are_you_sure: Ви впевнені?
   problem: Не вдалось зберегти зміни через помилки
+  submit: Готово
+  cancel: Скасувати
+  profitably: Вигідно
+  profitably_text: Наші тести абсолютно безкоштовні! Створіть свій перший тест вже зараз!
+  comfortably: Зручно
+  comfortably_text: Ви можете створити свій власний тест, самостійно обираючи тему та складність питань!
+  fast: Швидко
+  fast_text: За допомогою наших тестів Ви можете швидко перевірити свої знання або знання Ваших учнів з різних дисциплін!
+  why_quizland: Чому Квізланд?
```

Comment:
put under `home.home` all translations that belongs to home

to avoid having `home.home` embeddedness, I'd recommend to name the action `index`